safe_landing_planner/src/nodes/waypoint_generator.cpp

```diff
@@ -102,15 +130,25 @@ usm::Transition WaypointGenerator::runGoTo() {
     landing_radius_ = 0.5f;
     yaw_setpoint_ = avoidance::nextYaw(position_, goal_);
   }
+
   publishTrajectorySetpoints_(goal_, velocity_setpoint_, yaw_setpoint_, yaw_speed_setpoint_);
   ROS_INFO("\033[1;32m [WGN] goTo %f %f %f - %f %f %f \033[0m\n", goal_.x(), goal_.y(), goal_.z(),
            velocity_setpoint_.x(), velocity_setpoint_.y(), velocity_setpoint_.z());
   altitude_landing_area_percentile_ = landingAreaHeightPercentile(80.f);
-  std::fill(can_land_hysteresis_.begin(), can_land_hysteresis_.end(), 0.f);
+  can_land_hysteresis_matrix_.fill(0.0f);
 
   ROS_INFO("[WGN] Landing Radius: xy  %f, z %f ", (goal_.topRows<2>() - position_.topRows<2>()).norm(),
            fabsf(position_.z() - altitude_landing_area_percentile_));
 
+  if (found_land_area_in_grid_) {
+    landing_radius_ = 0.5f;
```

Comment:
Is this correct? Do we always want to use 0.5?

Reply:
When doing the grid evaluation and spiral pattern we need to be much more strict than with normal waypoints. The default acceptance radius is 2m. How would you feel about having it on a different variable?